community/planning/cylinder_jwt_authentication.md

```diff
@@ -0,0 +1,140 @@
+# Cylinder JWT Authentication
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+[summary]: #summary
+
+Cylinder JWT authorization for Splinter's REST APIs provides secure
+authentication for non-browser and non-user Splinter clients. This
+authentication method will be used by the Splinter CLIs and integration
+components.
+
+## Motivation
+[motivation]: #motivation
+
+Splinter's REST API provides routes for the Admin Service, Scabbard, Biome, and
+other Splinter components. Because it is desirable that the REST API be
+accessible to a variety of applications, it is important to have a robust
+authentication and authorization system implemented. While
+[OAuth2 support]({% link community/planning/oauth2_rest_api_authentication.md %})
+provides standards-based authentication for browser applications, Cylinder JWT
+support will enable non-user and non-browser clients to easily authenticate with
+the REST API.
+
+Cylinder JWTs are convenient because Cylinder is already an important part of
+the Splinter ecosystem; it is the library used to sign and verify signatures by
+the Splinter daemon itself as well as the Splinter CLIs. This makes it a prime
+candidate for REST API authentication. Additionally, Cylinder JWTs require no
+logins, handshakes, or other setup processes. They are self-signed and
+independently verifiable, which makes them both versatile and easy-to-use.
+
+## Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The Cylinder library provides all the tools necessary to create and verify
+Cylinder JWTs.
+
+### Creating a Cylinder JWT
+[creating-a-cylinder-jwt]: #creating-a-cylinder-jwt
+
+Splinter clients will create JWTs with the following steps:
+
+1. Load a Cylinder `PrivateKey` from the filesystem or another source
+1. Initialize a signing `Context` (such as the Secp256k1 context) that matches
```

Comment:
Looks like the numbering is off here

Reply:
Markdown handles the numbering automatically; using `1.` for every entry would allows us to change the order or insert items without having to mess with the numbering.